### PR TITLE
Add crawler detection page

### PR DESCRIPTION
**Pull request by sammacbeth**
Basic test page to allow tracker-radar-collector to test if it can be detected as a crawler.

### CODE REVIEW

File: crawler/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta encoding='utf-8'/>
+    <title>Headless detector</title>
+  </head>
+
+  <body>
```
Comment (kdzwinel) [inline crawler/index.html:8]: Can you please add breadcrumbs and a description like we have on other pages?

header

Sth like "This page collects various tests for detecting headless browsers."

File: crawler/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                if (result) {
+                    fetch(`./fail.json?test=${test}&${detectValue}`);
+                }
+            }
+
+            displayResult('userAgent', /HeadlessChrome/.test(window.navigator.userAgent), window.navigator.userAgent);
+            displayResult('plugins', navigator.plugins.length == 0, navigator.plugins.length);
+            displayResult('languages', navigator.languages == "", navigator.languages);
+            displayResult('webdriver', !!navigator.webdriver, navigator.webdriver)
+            displayResult('window.chrome', !window.chrome || !window.chrome.runtime, JSON.stringify(window.chrome))
```
Comment (kdzwinel) [inline crawler/index.html:41]: FYI there is no `window.chrome.runtime` in Chrome Canary (90) and my plugins are in fact empty. Probably not a big deal, but shows that those tests are somehow not reliable.

File: crawler/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            const ul = document.getElementById('results');
+            const template = document.getElementById('result-row');
+            
+            function displayResult(test, result, detectValue) {
+                const listEntry = template.content.cloneNode(true);
+                listEntry.querySelector('.key').textContent = test;
+                listEntry.querySelector('.value').textContent = result ? 'fail' : 'pass';
+                ul.appendChild(listEntry)
+                if (result) {
+                    fetch(`./fail.json?test=${test}&${detectValue}`);
```
Comment (kdzwinel) [inline crawler/index.html:33]: I like this :) I'd add a note that this is used for easy access to test results from the crawler's output.

**Review by kdzwinel — CHANGES_REQUESTED** [review]
Very neat! Could you please add a link to this page from https://github.com/duckduckgo/privacy-test-pages/blob/gh-pages/index.html ? We probably need "Other" category because it doesn't seem to fit anywhere else.

File: crawler/index.html (hunk 2 above)
Comment (sammacbeth) [reply crawler/index.html:41]: The articles around this also stress that some of these methods only work with certain versions. Having this also as a webpage means we can also test against normal chrome, and remove tests which no longer pass for real browsers.

**Comment by sammacbeth** [comment]
Thanks @kdzwinel - sorry I missed the docs on making a new page. I've updated the page to match the template and address your other comments.

**Review by kdzwinel — APPROVED** [review]
No worries and thanks again for adding this page!

**Pull request merged**